src/main/java/Event.java

```diff
@@ -0,0 +1,45 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+    protected LocalDateTime time;
+
+    /**
+     * Creates an Event object.
+     * 
+     * @param description The description of the event.
+     * @param time The time of the event.
+     */
+    public Event(String description, LocalDateTime time) {
+        super(description);
+        this.time = time;
+    }
+
+    /**
+     * Returns the message to be saved in the hard disk.
+     * 
+     * @return The string representation of the task in the local file.
+     */
+    @Override
+    public String writeMessage() {
+        String done = "";
```

Comment:
I think it is redundant to assign "done" to an empty string. You can just leave it as variable declaration only.